client/src/components/Users/Auth/SetNewPassword.tsx

```diff
@@ -10,53 +10,135 @@ import PasswordInput from './PasswordInput';
 import { UserContext } from '../../../providers';
 import { APP_API_BASE_URL } from '../../../configs';
 
+import SetNewPasswordImg from '../../../assets/set-new-password.svg';
+
 const useStyles = makeStyles((theme: Theme) => {
-  const xPadding = 12;
+  // const xPadding = 12;
   const yPadding = 6;
   const yMargin = 8;
 
   return {
     paper: {
-      maxWidth: 821 - theme.spacing(xPadding),
+      width: '60%',
+      // maxWidth: 821 - theme.spacing(xPadding),
       maxHeight: 732 - theme.spacing(yPadding),
       borderRadius: 10,
       marginTop: theme.spacing(yMargin),
       marginBottom: theme.spacing(yMargin),
-      paddingTop: theme.spacing(yPadding),
+      // paddingTop: theme.spacing(yPadding),
       paddingBottom: theme.spacing(yPadding),
-      paddingLeft: theme.spacing(xPadding),
-      paddingRight: theme.spacing(xPadding),
+      paddingLeft: 84,
+      paddingRight: 130,
+      // paddingLeft: theme.spacing(xPadding),
+      // paddingRight: theme.spacing(xPadding),
       margin: 'auto',
     },
-    header: { fontWeight: 'bold', marginBottom: 68 },
-    button: {
-      borderRadius: 0,
-      height: 62,
-      textTransform: 'none',
+    form: {
+      width: '100%',
+      paddingTop: 74,
+    },
+    ctaContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-end',
+      gap: 28,
+      paddingTop: 50,
+    },
+    ctaBtn: {
+      borderRadius: 8,
+      color: 'white',
+      height: 40,
+      padding: '8px 20px',
+    },
+    container: {
+      display: 'flex',
+      flexDirection: 'row',
+      paddingTop: 134,
+      paddingBottom: 160,
+    },
+    imgContainer: {
+      display: 'block',
+      maxWidth: 446,
+      minWidth: 200,
+      '@media (max-width:1050px)': {
+        display: 'none',
+      },
+    },
+    setNewPasswordImg: {
+      width: '100%',
     },
   };
 });
 
+const PASSWORD_MIN_LENGTH = 8;
+
+const ERRORS = {
+  PASSWORD_MIN_LENGTH: `Password needs to be at least ${PASSWORD_MIN_LENGTH} characters.`,
+  PASSWORD_MISMATCH: 'Passwords do not match.',
+};
+
 function SetNewPassword() {
   const RESOURCE_URL = `${APP_API_BASE_URL}/auth/users`;
   const classes = useStyles();
+  const sublabel = `(${PASSWORD_MIN_LENGTH} or more characters)`;
 
   const [password, setPassword] = React.useState<string>('');
+  const [confirmPassword, setConfirmPassword] = React.useState<string>('');
+
   const [error, setError] = React.useState<string | null>(null);
+  const [confirmPasswordError, setConfirmPasswordError] = React.useState<string | null>(null);
 
   const handleChange = (evt: React.ChangeEvent<HTMLInputElement>): void => {
     setPassword(evt.target.value);
   };
 
+  const handleBlurPassword = (): void => {
+    if (password.length < PASSWORD_MIN_LENGTH) {
```

Comment:
I think this should be 
```suggestion
    if (password.length <= PASSWORD_MIN_LENGTH) {
```

Reply:
You're right. Thanks for verifying.